src/modules/feed/containers/FeedPage.js

```diff
@@ -1,19 +1,76 @@
 import { h, Component } from "preact";
+import { connect } from "preact-redux";
+
 import { Page } from "src/components/Page";
 import { Header } from "src/components/Header";
+import Transaction from "../components/Transaction";
 import I18n from "src/config/i18n";
+import styles from "./FeedPage.scss";
+
+import {
+  fetchAllTransactions,
+  likeTransaction,
+  unLikeTransaction
+} from "../actions";
 
 export class FeedPage extends Component {
-  render() {
+  componentWillMount() {
+    this.props.fetchTransactions(this.props.user.apiToken, this.props.user.id);
+  }
+
+  render({ transactions, user, like, unLike }) {
+    const likeTransaction = transactionId => {
+      like(user.apiToken, user.id, transactionId);
+    };
+
+    const unLikeTransaction = transactionId => {
+      unLike(user.apiToken, user.id, transactionId);
+    };
+
     return (
       <Page>
         <Header>
           <h1>{I18n.t("feed.title")}</h1>
         </Header>
         <main />
+        <ul class={styles.transactionList} id="transactionList">
+          {transactions.map(transaction => {
+            let likeAction;
+            transaction.voted
+              ? (likeAction = unLikeTransaction)
+              : (likeAction = likeTransaction);
+
+            return (
+              <li key={transaction.id}>
+                <Transaction
+                  transaction={transaction}
+                  likeAction={likeAction}
+                />
+              </li>
+            );
+          })}
+        </ul>
       </Page>
     );
   }
 }
 
-export default FeedPage;
+const mapStateToProps = state => ({
+  user: state.authentication.user,
+  transactions: state.feed.transactions
+});
+
+const mapDispatchToProps = dispatch => {
```

Comment:
If the names are kept the same, you could even use:

```javascript
const mapDispatchToProps = {
  fetchAllTransations,
  likeTransaction,
  unLikeTransaction
}
```

and it will be wrapped around dispatch call internally.